src/release/breaking-changes/notifications.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Removing Notification.visitAncestor
+description: Notifications only traverse parents that are notification
+listeners.
+---
+
+## Summary
+
+Notifications are more efficient by traversing only parents that
+are notification listeners.
+
+## Context
+
+The notification API traversed the element tree in order to locate a
+notification receiver. This led to some unfortunate performance
+characteristics:
+
+  * If there was no receiver for a given notification type, the entire element
+    tree above the notification would be traversed and type checked.
+  * For multiple notifications in a given frame (which is common for scroll
+    views) we ended up traversing the element tree multiple times.
+
+If there were multiple or nested scroll views on a given page, the situation was
+worsened significantly - each scroll view would dispatch multiple notifications
+per frame. For example, in the Dart/Flutter Devtools flamegraph page, we found
+that about 30% of CPU time was spent dispatching notifications.
+
+In order to reduce the cost of dispatching notifications, we have changed
+notification dispatch so that it only visits parents that are notification
+listeners, reducing the number of elements visited per frame.
+
+However, the old notification system exposed the fact that it traversed
+each element as part of its API via `Notification.visitAncestor`. This
+method is no longer supported as we no longer visit all ancestor elements.
+
+## Description of change
+
+`Notification.visitAncestor` has been removed. Any classes which extend
+`Notification` should no longer override this method.
+
+** If `Notification.visitAncestor` has not been overriden, then no changes are
```

Comment:
If you don't implement a custom Notification that overrides... maybe?

Reply:
done